code/__DEFINES/dcs/signals.dm

```diff
@@ -726,3 +726,7 @@
 #define COMSIG_XENO_TURF_CLICK_CTRL "xeno_turf_click_alt"
 ///from monkey CtrlClickOn(): (/mob)
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"
+
+///SSalarm signals
```

Comment:
Make this a `//` not a `///` otherwise it will define only `COMSIG_TRIGGERED_ALARM ` as `SSalarm signals` in the doc, which will look weird.

Not necassary because we dont enforce doc here, but it would be nice